<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectionViewMain</class>
 <widget class="QMainWindow" name="InspectionViewMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLoad_Mean_Curvature">
   <property name="text">
    <string>Mean Curvature</string>
   </property>
   <property name="toolTip">
    <string>Load Mean Curvature</string>
   </property>
  </action>
  <action name="actionLoad_Shape_Index">
   <property name="text">
    <string>Shape Index</string>
   </property>
   <property name="toolTip">
    <string>Load Shape Index</string>
   </property>
  </action>
  <action name="actionLoad_Colorimetric">
   <property name="text">
    <string>Colorimetric Info</string>
   </property>
   <property name="toolTip">
    <string>Loads the L channel of the Lab space</string>
   </property>
  </action>
  <action name="actionMinimal_Bounding_Box">
   <property name="text">
    <string>Minimal Bounding Box</string>
   </property>
   <property name="toolTip">
    <string>Load Minimal Bounding Box</string>
   </property>
  </action>
  <action name="actionLoad_SDF">
   <property name="text">
    <string>Shape Diameter Function</string>
   </property>
   <property name="toolTip">
    <string>Load Shape Diameter Function</string>
   </property>
  </action>
  <action name="actionLoadFacets">
   <property name="text">
    <string>Load Facets</string>
   </property>
   <property name="toolTip">
    <string>Load Facets</string>
   </property>
  </action>
  <action name="actionLoad_Annotations">
   <property name="text">
    <string>Load Annotations</string>
   </property>
   <property name="toolTip">
    <string>Load annotations</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
